lib/jsonite/lets_proxy.rb

```diff
@@ -0,0 +1,23 @@
+class Jsonite
+  class LetsProxy < BasicObject
+
+    undef_method :==, :!, :!=
+
+    def initialize object, context, lets = {}
+      @__object, @__context, @__lets = object, context, lets
```

Comment:
Ah, and they `_*` variables refer to the actual names. It's been awhile since I've thought about HAL :flushed:

I'm good with `__@v__`.